libflatsurf/flatsurf/flat_triangulation.hpp

```diff
@@ -36,7 +36,7 @@
 
 namespace flatsurf {
 
-// A triangulated translation surface. For most purposes this is the central
+// A triangulated half-dilation surface. For most purposes this is the central
```

Comment:
Do you really want "half-dilation" here and not the more general "similarity"? Half-dilation means that paired edges must have associated vectors `v` and `a v` with a nonzero real.

Reply:
Ok. Let's do similarity of course.